HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -381,6 +384,11 @@ namespace WPEFramework
 	   {
 	       HdmiCecSink::_instance->sendDeviceUpdateInfo(header.from.toInt());
 	   }
+
+           if((header.from.toInt() == LogicalAddress::AUDIO_SYSTEM) && (HdmiCecSink::_instance->m_audioDevicePowerStatusRequested)) {
```

Comment:
why you limiting power status to audio device specific only ...that check can be done on the caller side if you are not interested on other devices power status